packages/api/NONCE

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Should remove this before merge, but it's required to trigger the api workflow path selector